linkerd/app/core/src/profiles.rs

```diff
@@ -79,11 +86,16 @@ where
     type Stream = Rx;
 
     fn get_routes(&self, dst: &NameAddr) -> Option<Self::Stream> {
-        let (tx, rx) = mpsc::channel(1);
+        if !self.suffixes.iter().any(|s| s.contains(dst.name())) {
+            debug!("name not in profile suffixes");
```

Comment:
should this include what the name was, or is this happening inside a span that already includes that?

Reply:
it's safe to assume all of that metadata will be on the context if it needs to be.